pkg/sql/opt/ordering/inverted_join_test.go

```diff
@@ -55,9 +55,9 @@ func TestInvertedJoinProvided(t *testing.T) {
 		t.Fatal(err)
 	}
 	tn = tree.NewUnqualifiedTableName("t2")
-	tab = md.AddTable(tc.Table(tn), tn)
+	inputTab := md.AddTable(tc.Table(tn), tn)
```

Comment:
I had to create a new variable rather than reuse `tab` so that the reference to `tab` in the `InvertedJoinPrivate` below is the correct table—the one with the inverted index.